low-level/files/opt/seagate/sspl/conf/sample_global_cortx_config.yaml

```diff
@@ -13,48 +13,40 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
-[SYSTEM_INFORMATION]
-rack_id=001
-site_id=001
-cluster_id=001
-node_id=001
-product=LDR_R2
-
-[SSPL-TESTS_SETTING]
-modules=RabbitMQingressProcessorTests, RabbitMQegressProcessor
-
-[RABBITMQEGRESSPROCESSOR]
-virtual_host=SSPL
-queue_name=actuator-req-queue
-exchange_name=sspl-in
-routing_key=actuator-req-key
-username=sspluser
-password=sspl4ever
-message_signature_username=sspl-ll
-message_signature_token=ALOIUD986798df69a8koDISLKJ282983
-message_signature_expires=3600
-primary_rabbitmq_host=localhost
-
-[RABBITMQINGRESSPROCESSORTESTS]
-virtual_host=SSPL
-queue_name=sensor-queue
-exchange_name=sspl-out
-routing_key=sensor-key
-username=sspluser
-password=sspl4ever
-primary_rabbitmq_host=localhost
-
-[RAIDSENSOR]
-monitor=true
-
-[REALSTORSENSORS]
-monitor=true
-
-[NODEHWSENSOR]
-monitor=true
-
-[SYSTEMDWATCHDOG]
-monitor=true
-
-[RABBITMQCLUSTER]
-cluster_nodes=localhost
+cluster:
+   cluster_id: 'CortxCluster001'
+   product: LDR_R2
+   server_nodes:
+   srvnode_1:
+      operating_system: centos7
```

Comment:
OS needs to be read from /etc/system-release by sspl, so even if we keep in this sample file, Provisioner wont support this later and our code may break. So better remove this from here too.
This applies to any config which we are keeping here and Provisioner sheet has marked it red(means provisioner wont support).

Reply:
removed.